dataikuapi/dss/flow.py

```diff
@@ -338,6 +338,22 @@ def add_item(self, obj):
         self._raw = self.client._perform_json("POST", "/projects/%s/flow/zones/%s/items" % (self.flow.project.project_key, self.id),
                                   body=self.flow._to_smart_ref(obj))
 
+    def add_items(self, items):
+        """
+        Adds items to this zone.
+
+        The items will automatically be moved from its existing zone. Additional items may be moved to this zone
+        as a result of the operation (notably the recipe generating `obj`).
+
+        :param object items: A :class:`dataikuapi.dss.dataset.DSSDataset`, :class:`dataikuapi.dss.managedfolder.DSSManagedFolder`,
+                           or :class:`dataikuapi.dss.savedmodel.DSSSavedModel` to add to the zone
```

Comment:
A few things here should be updated now that it's a list of items, probably operation -> operations, that phrase about the recipe, and the param description should be updated to say "a list of ..."